arangod/V8Server/v8-collection.cpp

```diff
@@ -1147,12 +1147,7 @@ static void JS_LoadVocbaseCol(v8::FunctionCallbackInfo<v8::Value> const& args) {
     TRI_V8_THROW_EXCEPTION_INTERNAL("cannot extract collection");
   }
 
-  auto res = methods::Collections::load(vocbase, collection);
-
-  if (res.fail()) {
-    TRI_V8_THROW_EXCEPTION(res);
-  }
-
+  // load is a no-op starting with ArangoDB 3.9
```

Comment:
It is deprecated since 3.9 already, and a no-op since then: https://github.com/arangodb/docs/blob/main/3.9/release-notes-api-changes39.md#javascript-api